tools/lib/provision_inner.py

```diff
@@ -248,7 +248,11 @@ def main(options: argparse.Namespace) -> int:
             print("No need to run `scripts/setup/configure-rabbitmq.")
 
         dev_template_db_status = DEV_DATABASE.template_status()
```

Comment:
It would be nice to skip the `template_status` step if you don't need it, but I suppose it's too inexpensive to worry about if it only runs needlessly on CI.

Reply:
Updated with this applied.